infra/aws/scripts/helpers.ps1

```
@@ -133,19 +133,25 @@ function Copy-ToRemote {
 
 # -- Set-RemoteEnvOverlay -----------------------------------------------------
-# Reads a KEY=VALUE overlay file (lines beginning with # are ignored) and
-# applies each entry to the remote .env file at $RemoteEnvFile using sed:
+# Reads a KEY=VALUE overlay and applies each entry to the remote .env file at
+# $RemoteEnvFile using sed:
 #   - If the key exists: the value is replaced in-place.
 #   - If the key is absent: the key=value line is appended.
+#
+# Supply EITHER -OverlayFile (path to a local .tpl file) OR -OverlayContent
+# (a multi-line string of KEY=VALUE pairs).  Lines beginning with # are ignored.
 function Set-RemoteEnvOverlay {
```

PSScriptAnalyzer (1.25.0)

[warning] Function 'Set-RemoteEnvOverlay' has verb that could change system state. Therefore, the function has to support 'ShouldProcess'.

(PSUseShouldProcessForStateChangingFunctions)